tests/ui/unsized-locals/suggest-borrow.stderr

```diff
@@ -44,10 +44,10 @@ LL |     let x: &[u8] = &vec!(1, 2, 3)[..];
    |            +
 
 error[E0277]: the size for values of type `[u8]` cannot be known at compilation time
-  --> $DIR/suggest-borrow.rs:4:9
+  --> $DIR/suggest-borrow.rs:4:19
    |
 LL |     let x: [u8] = &vec!(1, 2, 3)[..];
-   |         ^ doesn't have a size known at compile-time
+   |                   ^^^^^^^^^^^^^^^^^^ doesn't have a size known at compile-time
```

Comment:
This is weird: it should point at the type, not the init expr. If I modify the test to make this the only thing here, then *it does the right thing*. I'm not *too* concerned because of the close locality but it is technically incorrect (as the expr has type `&[u8]`, *the binding* is `[u8]`).